addons/pinniped/config-controller/controllers/v1_controller.go

```diff
@@ -135,9 +136,11 @@ func (c *PinnipedV1Controller) reconcileAddonSecret(ctx context.Context, cluster
 	// also check if cluster is scheduled for deletion, if so, delete addon secret on mgmt cluster
 	if pinnipedInfoCM.Data == nil || !cluster.GetDeletionTimestamp().IsZero() {
 		log.V(1).Info("deleting secret")
-		if err := c.reconcileDelete(ctx, client.ObjectKeyFromObject(cluster), log); err != nil {
+		if err := c.reconcileDelete(ctx, secretNameFromClusterName(client.ObjectKeyFromObject(cluster)), log); err != nil {
+			log.Info("failed to delete, error")
 			return err
 		}
+		log.Info("successfully deleted secret")
```

Comment:
We also log when we have finished deleting the secret in `reconcileDelete`, so this seems unnecessary.  I made it a V1 in `reconcileDelete` because otherwise the log output was pretty excessive, but I'd be ok moving it back to a V0.

Reply:
happy to remove.